jobs/models.py

```diff
@@ -100,7 +100,11 @@ def __str__(self):
         return 'Job Listing #{}'.format(self.pk)
 
     def save(self, **kwargs):
-        self.location_slug = slugify('%s %s %s' % (self.city, self.region, self.country))
+        location_str = ''
+        for location_part in (self.city, self.region, self.country):
+            if location_part is not None:
+                location_str = ' '.join(location_str, location_part)
```

Comment:
same as above. also, `' '.join(location_str, location_part)` should be `' '.join([location_str, location_part])`

Thanks!